airflow/executors/celery_executor.py

```diff
@@ -41,6 +41,8 @@
 
 CELERY_SEND_ERR_MSG_HEADER = 'Error sending Celery task'
 
+SEND_TASK_TIMEOUT = conf.getint('celery', 'SEND_TASK_TIMEOUT')
```

Comment:
add ,fallback=2

Reply:
fallback is not needed because `AirflowConfigParser.get()` looks in default_airflow.cfg if the entry is not found. I set it to 2 in `default_airflow.cfg` so we don't need explicit fallback here.